lib/process-services-cloud/src/lib/form/components/widgets/date/date-cloud.widget.spec.ts

```diff
@@ -424,15 +425,15 @@ describe('DateWidgetComponent', () => {
                 fixture.detectChanges();
                 await fixture.whenStable();
 
-                const expectedMaxValueString = '2022-07-30';
+                const currentDate = new Date();
+                const expectedMaxValueString = DateFnsUtils.formatDate(addDays(currentDate, 8), DATE_FORMAT_CLOUD);
```

Comment:
this makes no sense, the expected value was already provided, you don't need to convert anything